Geomap does not work in NAV 5.0.1

**Describe the bug**

Geomap loads but is empty.  The following stacktrace is found in the logs:

```
Tue Dec 10 16:36:29.931921 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403] [Tue Dec 10 16:36:29 2019] [ERROR] [pid=21900 django.request] Internal Server Error: /geomap/open/data
[Tue Dec 10 16:36:29.931979 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403] Traceback (most recent call last):
[Tue Dec 10 16:36:29.931988 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]   File "/usr/local/lib/python3.7/site-packages/django/core/handlers/exception.py", line 41, in inner
[Tue Dec 10 16:36:29.931995 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]     response = get_response(request)
[Tue Dec 10 16:36:29.932003 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]   File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
[Tue Dec 10 16:36:29.932010 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]     response = self.process_exception_by_middleware(e, request)
[Tue Dec 10 16:36:29.932017 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]   File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
[Tue Dec 10 16:36:29.932025 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]     response = wrapped_callback(request, *callback_args, **callback_kwargs)
[Tue Dec 10 16:36:29.932032 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]   File "/usr/local/lib/python3.7/site-packages/nav/web/geomap/views.py", line 163, in data
[Tue Dec 10 16:36:29.932040 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]     do_fetch_data)
[Tue Dec 10 16:36:29.932047 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]   File "/usr/local/lib/python3.7/site-packages/nav/web/geomap/views.py", line 208, in get_formatted_data
[Tue Dec 10 16:36:29.932055 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]     features = create_features(variant, graph, do_create_edges)
[Tue Dec 10 16:36:29.932062 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]   File "/usr/local/lib/python3.7/site-packages/nav/web/geomap/features.py", line 63, in create_features
[Tue Dec 10 16:36:29.932069 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]     nodes = [node_feature_creator(n) for n in graph.nodes.values()]
[Tue Dec 10 16:36:29.932076 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]   File "/usr/local/lib/python3.7/site-packages/nav/web/geomap/features.py", line 63, in <listcomp>
[Tue Dec 10 16:36:29.932084 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]     nodes = [node_feature_creator(n) for n in graph.nodes.values()]
[Tue Dec 10 16:36:29.932091 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]   File "/usr/local/lib/python3.7/site-packages/nav/web/geomap/features.py", line 156, in create_node_feature
[Tue Dec 10 16:36:29.932115 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]     create_node_popup(node, popup_template),
[Tue Dec 10 16:36:29.932122 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]   File "/usr/local/lib/python3.7/site-packages/nav/web/geomap/features.py", line 164, in create_node_popup
[Tue Dec 10 16:36:29.932130 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]     content = popup_template.render({'place': node.properties})
[Tue Dec 10 16:36:29.932136 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]   File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 203, in render
[Tue Dec 10 16:36:29.932144 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403]     with context.render_context.push_state(self):
[Tue Dec 10 16:36:29.932150 2019] [wsgi:error] [pid 21900:tid 140253048252160] [remote 192.168.0.1:10403] AttributeError: 'dict' object has no attribute 'render_context'

```

**To Reproduce**

To reproduce, at least one IP Device located in a Room with a set geolocation must exist in the NAV database.

**Expected behavior**

Expected to see all rooms at their locations/coordinates


**Environment (please complete the following information):**
 - Browser: Firefox 70.0.1 on Ubuntu 16.04 LTS
 - OS on the NAV server: Ubuntu 16.04 LTS
 - NAV version installed: 5.0.1
 - Method of installation: source

--- a/python/nav/web/geomap/features.py
+++ b/python/nav/web/geomap/features.py
@@ -34,7 +34,7 @@ import os
 import logging
 from functools import partial
 
-from django import template
+from django.template import engines
 
 from nav.config import open_configfile
 from nav.web.geomap.conf import get_configuration
@@ -81,14 +81,15 @@ def load_popup_template(filename):
     :param filename: name of a configuration file containing template source
                      (relative to the geomap configuration directory)
 
-    :returns: A django.template.Template object.
+    :returns: A django.template.backends.django.Template object.
     """
     if filename is None:
         return None
     filename = os.path.join('geomap', filename)
     with open_configfile(filename) as afile:
         content = afile.read()
-    return template.Template(content)
+    django_engine = engines['django']
+    return django_engine.from_string(content)
 
 
 def apply_indicator(ind, properties):

